install/installer/pkg/common/storage.go

```diff
@@ -51,7 +51,7 @@ func StorageConfig(context *RenderContext) storageconfig.StorageConfig {
 				Endpoint:            context.Config.ObjectStorage.S3.Endpoint,
 				AccessKeyIdFile:     filepath.Join(storageMount, "accessKeyId"),
 				SecretAccessKeyFile: filepath.Join(storageMount, "secretAccessKey"),
-				Secure:              true,
+				Secure:              !context.Config.ObjectStorage.S3.AllowInsecureConnection,
```

Comment:
I am imagining this would become `true` by default because in Go `bool` has default value `false`. Testing just to be sure

Reply:
seems to be the case!

Reply:
Yup. I added two new golden file tests to prove it as well